db/migrate/20200427055907_remove_admin_from_users.rb

```diff
@@ -1,5 +0,0 @@
-class RemoveAdminFromUsers < ActiveRecord::Migration
```

Comment:
Why did you remove the migration file? After removing the file how did you test?